extension-androidauto/src/main/java/com/mapbox/maps/extension/androidauto/DefaultMapboxCarMapGestureHandler.kt

```diff
@@ -0,0 +1,120 @@
+package com.mapbox.maps.extension.androidauto
+
+import androidx.car.app.SurfaceCallback
+import com.mapbox.common.Logger
+import com.mapbox.maps.CameraOptions
+import com.mapbox.maps.MapboxExperimental
+import com.mapbox.maps.MapboxMap
+import com.mapbox.maps.ScreenCoordinate
+import com.mapbox.maps.plugin.animation.camera
+
+/**
+ * This class contains the default map gestures. It Handles the gestures received from
+ * [SurfaceCallback] and applies them to the [MapboxMap] camera. If you would like to customize
+ * the map gestures, use [MapboxCarMap.setGestureHandler].
+ */
+@MapboxExperimental
+open class DefaultMapboxCarMapGestureHandler : MapboxCarMapGestureHandler {
+
+  /**
+   * @see [MapboxCarMapGestureHandler.onScroll]
+   * @see [SurfaceCallback.onScroll] for instructions to enable.
+   *
+   * @param mapboxCarMapSurface loaded and ready car map surface
+   * @param distanceX the distance in pixels along the X axis
+   * @param distanceY the distance in pixels along the Y axis
+   *
+   * @return true when the scroll call was handled, false will trigger the default handler
+   */
+  override fun onScroll(
+    mapboxCarMapSurface: MapboxCarMapSurface,
+    visibleCenter: ScreenCoordinate,
+    distanceX: Float,
+    distanceY: Float
+  ): Boolean {
```

Comment:
If we go with open class approach, wonder do we still need to return a boolean, as user can decide whether to call `super.onScroll()` to invoke the default handler.

Reply:
The reason I left it in was for **forward compatibility**.

For example, `onFling` returns `false`. One day it will handle fling and it can return `true` without breaking the interface contract.

This matters when extending `DefaultMapboxCarMapGestureHandler` and you call `super.onScroll`, `super.onScale` or `super.onFling`. With a boolean return you can say something like:

``` kotlin
  override fun onFling(
    map: MapboxCarMapSurface,
    velocityX: Float,
    velocityY: Float
  ) {
    if (!super.onFling(map, velocityX, velocityY)) {
      // trigger my own custom map fling
    } 
  }
```

Reply:
But agree it does not make sense for `MapboxCarMapGestureHandler`, so removing